src/utils.js

```diff
@@ -67,3 +69,37 @@ export function forNestedRelations(model, nestedRelations, fn) {
 
     });
 }
+
+// stale request utils
+
+/**
+ * Generate a uuid, and add it to the requestQ
+ * @param {string[]} requestQ a list of request ids
+ * @returns {uuidv4} uuid for the request
+ */
+export function generateRequestId() {
```

Comment:
I think you might as well remove this function: it simply calls a standard library function, without adding anything.